### PR TITLE
utils/github: delete redundant `GitHub.` and `GitHub::`

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

* Deleted occurrences of `GitHub.` and `GitHub::` that are redundant
* Replaced `api_errors` method with constant `API_ERRORS`

### CODE REVIEW

File: Library/Homebrew/dev-cmd/bump-formula-pr.rb [hunk 1]
```diff
@@ -395,7 +400,7 @@ def bump_formula_pr
           else
             exec_browser url
           end
-        rescue *GitHub.api_errors => e
+        rescue *GitHub.API_ERRORS => e
```
Comment (reitermarkus) [inline Library/Homebrew/dev-cmd/bump-formula-pr.rb:403]:
```suggestion
        rescue *GitHub::API_ERRORS => e
```

Comment (SeekingMeaning) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb:403]: Whoops, thanks!

**Comment by SeekingMeaning** [comment]
#8177 introduced a bug:

> * Also updates `inreplace_pairs` to check for `--write` in addition to `--dry-run`

`inreplace_pairs` already checked for `--write`, just not in the `if` statement

**Comment by SeekingMeaning** [comment]
Closing for now

**Pull request closed without merging**